package.json

```diff
@@ -19,6 +19,12 @@
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json",
+    "typeorm": "npm run build && npx typeorm -d ./dist/config/data-source.js",
+    "migration:generate": "npm run typeorm -- migration:generate",
+    "migration:create": "npm run typeorm -- migration:create",
+    "migration:run": "npm run typeorm -- migration:run",
+    "migration:show": "npm run typeorm -- migration:show",
+    "migration:revert": "npm run typeorm -- migration:revert",
```

Comment:
can ignore `npm run` here.

Reply:
seems like the `typeorm` is not a cli command, it's the command here at LINE 22. I misunderstanded the meaning.

Reply:
seems like the `typeorm` is not a cli command, it's the command here at LINE 22. I was misunderstanded the meaning, sorry.